cpp/include/nvtext/generate_ngrams.hpp

```diff
@@ -90,5 +90,36 @@ std::unique_ptr<cudf::column> generate_character_ngrams(
   cudf::size_type ngrams              = 2,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Hashes ngrams of characters within each string
```

Comment:
Should the description include what hashing algorithm is used in the implementation?